components/common/Avatar.tsx

```diff
@@ -4,51 +4,45 @@ import Image from "next/image";
 import { Skeleton } from "../shadcn/skeleton";
 import { Avatar as ShadcnAvatar } from "@/components/shadcn/avatar";
 import { web3ImgLoader } from "@/lib/web3-img-loader";
+import { cn } from "@/lib/tailwind";
 
 interface AvatarProps {
   uri: string;
+  className?: string;
 }
 
-export const HeaderAvatar: React.FC<AvatarProps> = ({ uri }) => {
+interface AvatarLoaderProps {
+  className?: string;
+}
+
+export function HeaderAvatar({ uri, className }: AvatarProps) {
   return (
-    <ShadcnAvatar className="h-7 w-7 sm:h-[30px] sm:w-[30px]">
-      <Image
-        alt="avatar"
-        width={30}
-        height={30}
-        src={uri}
-        loader={web3ImgLoader}
-      />
+    <ShadcnAvatar className={cn("h-7 w-7 sm:h-8 sm:w-8", className)}>
+      <Image alt="avatar" src={uri} loader={web3ImgLoader} fill />
```

Comment:
please don't use `fill`, this loads huge image even if the render is tiny

3K px for the top right avatar for example

<img width="908" alt="Screenshot 2025-03-12 at 05 17 11" src="https://github.com/user-attachments/assets/8ea9fd04-9eec-4fe7-b7c1-bff329024949" />